interface/super/edit_list.php

```diff
@@ -991,38 +1008,48 @@ function mysubmit() {
-            <a class="navbar-brand"
-               href="#"><?php echo xlt('Manage Lists'); ?></a>
-        </div>
-
-        <!-- Collect the nav links, forms, and other content for toggling -->
-        <div class="collapse navbar-collapse" id="navbar-list">
-            <ul class="nav navbar-nav">
-                <li><a href="#" data-toggle="modal"
-                       data-target="#modal-new-list"><i class="fa fa-plus"></i>&nbsp;<?php echo xlt('New List'); ?>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="deletelist" id="<?php echo attr($list_id); ?>">
-                        <i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?>
-                    </a>
-                </li>
-            </ul>
+    <input type="hidden" name="csrf_token_form" value="<?php echo attr(collectCsrfToken()); ?>" />
+    <input type="hidden" name="list_from" value="<?php echo attr($list_from);?>"/>
+    <input type="hidden" name="list_to" value="<?php echo attr($list_to);?>"/>
+    <nav class="navbar navbar-default navbar-fixed-top">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed"
+                        data-toggle="collapse" data-target="#navbar-list"
+                        aria-expanded="false">
+                    <span class="sr-only"><?php echo xlt('Toggle navigation'); ?></span>
+                    <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand"
+                   href="#"><?php echo xlt('Manage Lists'); ?></a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="navbar-list">
+                <ul class="nav navbar-nav">
+                    <li><a href="#" data-toggle="modal"
+                           data-target="#modal-new-list"><i class="fa fa-plus"></i>&nbsp;<?php echo xlt('New List'); ?>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="deletelist" id="<?php echo attr($list_id); ?>">
+                            <i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?>
+                        </a>
+                    </li>
+                </ul>
                 <input type="hidden" name="formaction" id="formaction">
                 <div class="form-group navbar-left">
                     <select name='list_id' class="form-control select-dropdown"
                             id="list_id">
                         <?php
+                        /*
+                         * Keep proper list name (otherwise list name changes according to
+                         * the options shown on the screen).
+                         */
+                        $list_id_container = attr($_GET["list_id_container"]);
```

Comment:
drop the attr(). see my comment I just posted in the prior PR.

Reply:
Hi @bradymiller . Thank you, erased.